source/agora/utils/Log.d

```diff
@@ -143,9 +143,9 @@ public class CircularAppender : Appender
 
     public static CircularAppender opCall ()
     {
-        static CircularAppender appender;
+        static CircularAppender!BufferSize appender;
```

Comment:
Protip: Inside a template, you can use the template name to refer to the instantiation. That's why you didn't have to change this function's return value.

Reply:
got it thanks